fastlane/lib/fastlane/actions/build_app.rb

```diff
@@ -5,7 +5,7 @@ module SharedValues
       DSYM_OUTPUT_PATH = :DSYM_OUTPUT_PATH
     end
 
-    class GymAction < Action
+    class BuildAppAction < Action
```

Comment:
How does renaming action files impact the doc generation? If somebody searches for `gym` will gym docs be available? We say "via gym", "via snapshot" - will users be able to understand what these are?

Reply:
Excellent question - looking into this now...

Reply:
Seems like we might be OK because of the changes in commit `d0b5408` ?